source/utilities.tex

```diff
@@ -3198,14 +3198,20 @@
   template<class T>
     class optional;                                                     // partially freestanding
 
+  // \ref{optional.optional.ref}, partial specialization of \tcode{optional} for lvalue reference types
   template<class T>
-    constexpr bool ranges::enable_view<optional<T>> = true;
+  class optional<T\&>;                                                  // partially freestanding
+
+  template<class T>
+  constexpr bool ranges::enable_view<optional<T>> = true;
   template<class T>
-    constexpr auto format_kind<optional<T>> = range_format::disabled;
+  constexpr auto format_kind<optional<T>> = range_format::disabled;
+  template<class T>
+  constexpr bool ranges::enable_borrowed_range<optional<T\&>> = true;
 
   template<class T>
-    concept @\defexposconcept{is-derived-from-optional}@ = requires(const T& t) {       // \expos
-      []<class U>(const optional<U>&){ }(t);
+    concept @\defexposconcept{is-derived-from-optional}@ = requires(const T\& t) {       // \expos
```

Comment:
We don't need / want quoting of ampersands inside codeblocks.   Also, these LaTeX changes were likely not asked for in the incoming paper.  Please avoid those drive-by "fixes" as much as possible when applying papers, and put them into separate pull requests.  This eases further review.

Reply:
Fixed along with 3 other stray escapes.